### ISSUE TITLE
viewer can not see cursor and latest edits in onlyoffice

### ISSUE THREAD

**Issue by nirajacharya2** [owncloud/ocis#10468]
## Describe the bug
latest edits and the mouse cursor are not shown in onlyoffice. if the file is locked onlyoffice won't show the latest version of a flie.

## Steps to reproduce
1. admin share `new.docx` with marie (viewer role)
2. marie opens `new.docx` with onlyoffice
3. admin edits `new.docx`

## Expected behavior
marie should be able to see the cursor and added text.
![image](https://github.com/user-attachments/assets/1191aefe-80d6-45df-b003-6ec92cfa2886)

## Actual behavior
marie cannot see the cursor and added text.
![image](https://github.com/user-attachments/assets/b12bec06-324e-48d4-9e3a-28828065be56)

## Setup
Please describe how you started the server and provide a list of relevant environment variables or configuration files.

<details>
<p>

```console
OCIS=6.6.1
onlyoffice=8.2.0
```

</p>
</details>

## Additional context

@ScharfViktor @micbar @jvillafanez @kobergj

**Comment by micbar** [comment]
This is a duplicate of #4570 which has been closed due to lack of progress.

@hodyroff @dj4oC Do you want to push this?

**Comment by hodyroff** [comment]
Yes. Checking the documentation this must be enabled with an additional sharing function.
https://helpcenter.onlyoffice.com/ONLYOFFICE-Editors/ONLYOFFICE-Document-Editor/HelpfulHints/CollaborativeEditing.aspx#:~:text=The%20Live%20Viewer%20mode%20is%20used%20to%20see,checkbox%20is%20active%20in%20the%20editor%27s%20Advanced%20Settings.
@jvillafanez @kobergj

**Comment by hodyroff** [comment]
In any case a missing feature in the integration, not a bug.

**Comment by hodyroff** [comment]
> @hodyroff this is a setting in the php connector app.
> 
> Is there any resource how to achieve that via WOPI?

We have to ask Onlyoffice ... the link just explained to me that its a normal behaviour or better that there are two different viewer behaviours.

**Comment by ghost** [comment]
duplicate of https://github.com/owncloud/ocis/issues/11473
